shared/chat/conversation/banner/container.js

```diff
@@ -51,6 +51,8 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
 export default compose(
   connect(mapStateToProps, mapDispatchToProps),
   branch(props => !props, renderNothing),
+  // $FlowIssue gets very confused here
```

Comment:
there's currently some general crappiness around recompose. I think the jist of it is:
We have a pattern where we use connect() to extract some props
We then use branch to render different types of components 
We don't usually whittle down the subset of props so each one gets its own set and instead usually send a superset to all of them. With this update of flow-typed it no longer likes that